TrailInsurance/RestClient+NewClaim.swift

```diff
@@ -67,13 +67,14 @@ extension RestClient {
 	///   - accountID: The ID of the master account.
 	func fetchMasterAccountForUser(onFailure failureHandler: @escaping RestFailBlock, completionHandler: @escaping (_ accountID: String) -> Void) {
 		let userID = UserAccountManager.shared.currentUserAccount!.accountIdentity.userId
-		let accountRequest = self.request(forQuery: "SELECT ID FROM Account WHERE ownerId = '\(userID)' AND isMaster__c = true LIMIT 1")
+		let accountRequest = self.request(forQuery: "SELECT Contact.accountID FROM User WHERE Id = '\(userID)' LIMIT 1")
```

Comment:
`Contact.AccountId`

Reply:
Since SOQL is not case sensitive outside of a where clause, this doesn't make a functional difference. Is there a reason you're pointing this out?

Reply:
Consistency with the real field name.

Reply:
A distinction without a difference, in this case. but I've made the requested change.